src/main/java/kr/sesac/aoao/server/item/controller/dto/GetItemInfoResponse.java

```diff
@@ -0,0 +1,20 @@
+package kr.sesac.aoao.server.item.controller.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class GetItemInfoResponse {
+	private final Long id;
+	private final String name;
+	private final int price;
+	private final int exp;
+
+	@Builder
+	public GetItemInfoResponse(Long id, String name, int price, int exp){
+		this.id = id;
+		this.name = name;
+		this.price = price;
+		this.exp = exp;
+	}
+}
```

Comment:
```suggestion
	public GetItemInfoResponse(Long id, String name, int price, int exp){
		this.id = id;
		this.name = name;
		this.price = price;
		this.exp = exp;
	}
}
```

저희 빌더 사용안하기로 했습니당 ㅎㅎ 
서비스단에서 어떻게 사용해야 할지 자세하게 적어놓았으니 참고해주세용